proxy/http3/Http3Transaction.cc

```diff
@@ -595,11 +623,17 @@ Http09Transaction::state_stream_closed(int event, void *data)
 
   switch (event) {
   case VC_EVENT_READ_READY:
+    if (this->_read_event == data) {
+      this->_read_event = nullptr;
+    }
```

Comment:
funny the way complier complains about `[[fallthrough]]` as here seems not to care (probably because it just `break`s)